src/backend/joanie/core/models/products.py

```diff
@@ -1139,6 +1140,42 @@ def withdraw(self):
 
         self.flow.cancel()
 
+    def flow_init(self, billing_address=None):
```

Comment:
The name is confusing. When you work on an order instance you can do `instance.flow.something()` but also `instance.flow_init()`. Changing it with something like `init_flow` is less error prone I think.